.github/workflows/upstream.yml

```diff
@@ -0,0 +1,59 @@
+name: Upstream reference checks
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-2.[5-9]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  kind-tests:
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: localhost:5000
+    strategy:
+      matrix:
+        # Run tests on oldest and newest supported OCP Kubernetes
+        # - OCP 4.5 runs Kubernetes v1.18
+        # KinD tags: https://hub.docker.com/r/kindest/node/tags
+        kind:
+          - 'v1.18.15'
+          - 'latest'
+        deployOnHub:
+          - 'true'
+          - 'false'
+
+    name: Upstream reference checks
+    steps:
+    - name: Checkout propagator
+      uses: actions/checkout@v2
+      with:
+        path: governance-policy-propagator
+        fetch-depth: 0 # Fetch all history for all tags and branches
+
+    - name: Checkout Policy Framework
+      uses: actions/checkout@v2
+      with:
+        path: governance-policy-framework
+        repository: stolostron/governance-policy-framework
+        ref: ${{ github.event.pull_request.base.ref }}
+
+    - name: Set up Go 1.17
+      uses: actions/setup-go@v2
+      id: go
+      with:
+        go-version: '1.17'
+
+    - name: Detect upstream references that should be replaced
+      working-directory: governance-policy-propagator
+      run: |
+        ./../governance-policy-framework/build/detect-upstream.sh -m "cat Makefile" -i "go list -mod readonly -m all" -q "deploy/manager/manager.yaml" -q "deploy/operator.yaml"
+
+    - name: Verify upstream references checking is working properly
```

Comment:
This can be removed since the tests are run on the Policy Framework CI. We can just trust that they work here.